js/web/lib/onnxjs/backends/webgl/ops/conv.ts

```diff
@@ -325,7 +326,8 @@ export class WebGLUnpackedConv extends Conv {
     const outputLayout = inferenceHandler.createTextureLayoutFromShape(outputShape);
     const initValue = (inputs.length < 3) ? '0.0' : '_B(b)';
     const sharedDim = im2colLayout.shape[3];
-    const blendEnabled = inferenceHandler.session.backend.glContext.isBlendSupported && !this.activation;
+    // const blendEnabled = inferenceHandler.session.backend.glContext.isBlendSupported && !this.activation;
```

Comment:
Remove this line or put a reason why we just disable blend.

Reply:
resolved.